cmd/kk/pkg/core/util/file.go

```diff
@@ -237,7 +237,6 @@ func Untar(src, dst string) error {
 				return err
 			}
 
-			fmt.Println(dstPath)
```

Comment:
I don't know if this line is needed to remove at present. So, I suggest keeping it.

Reply:
😀, I'll refresh the pr to keep this line.